# HG changeset patch
# User Ben Kelly <ben@wanderview.com>
# Date 1411498095 14400
#      Tue Sep 23 14:48:15 2014 -0400
# Node ID 69a64ce63c921648690ffc0259fd47bb0438d38e
# Parent f06ce234f21531cfc1b1b3c01123f3093ddfdf5a
Fix static constants again to satisfy windows compiler.

diff --git a/dom/cache/DBSchema.cpp b/dom/cache/DBSchema.cpp
--- a/dom/cache/DBSchema.cpp
+++ b/dom/cache/DBSchema.cpp
@@ -15,18 +15,18 @@
 #include "nsCOMPtr.h"
 #include "nsTArray.h"
 
 namespace mozilla {
 namespace dom {
 namespace cache {
 
 
-const int32_t DBSchema::kLatestSchemaVersion;
-const int32_t DBSchema::kMaxEntriesPerStatement;
+const int32_t DBSchema::kLatestSchemaVersion = 1;
+const int32_t DBSchema::kMaxEntriesPerStatement = 255;
 
 using mozilla::void_t;
 
 // static
 nsresult
 DBSchema::CreateSchema(mozIStorageConnection* aConn)
 {
   MOZ_ASSERT(aConn);
diff --git a/dom/cache/DBSchema.h b/dom/cache/DBSchema.h
--- a/dom/cache/DBSchema.h
+++ b/dom/cache/DBSchema.h
@@ -94,17 +94,17 @@ private:
                                       uint32_t aPos, int32_t aLen);
   static nsresult BindListParamsToQuery(mozIStorageStatement* aState,
                                         const nsTArray<EntryId>& aEntryIdList,
                                         uint32_t aPos, int32_t aLen);
 
   DBSchema() MOZ_DELETE;
   ~DBSchema() MOZ_DELETE;
 
-  static const int32_t kLatestSchemaVersion = 1;
-  static const int32_t kMaxEntriesPerStatement = 255;
+  static const int32_t kLatestSchemaVersion;
+  static const int32_t kMaxEntriesPerStatement;
 };
 
 } // namespace cache
 } // namespace dom
 } // namespace mozilla
 
 #endif // mozilla_dom_cache_DBSchema_h
